aptos-move/block-executor/src/errors.rs

```diff
@@ -3,9 +3,6 @@
 
 #[derive(Debug, PartialEq, Eq)]
 pub enum Error<E> {
-    /// Invariant violation that happens internally inside of scheduler, usually an indication of
-    /// implementation error.
-    InvariantViolation,
```

Comment:
Why is this error removed?

Reply:
Was never used - happy to bring it back whenever needed.